docs/source/tools_supported.rst

```diff
@@ -128,3 +130,23 @@ You can also re-use the same conversion script if you have multiple inputs to co
     mavis config \
         --external_conversion my_converted_input1 "my_convert_script.py my_input1.txt" \
         --external_conversion my_converted_input2 "my_convert_script.py my_input2.txt"
+
+
+
+.. _general-vcf-inputs:
+
+
+General VCF inputs
+-------------------
+
+Assuming that the tool outputting the VCF file follows standard conventions, then it is possible to use a general VCF 
+conversion that is not tool-specific. Given the wide variety in content for VCF files, MAVIS makes a number of
+assumptions and the VCF conversion may not work for all VCFs. In general MAVIS follows the `VCF 4.2 specification <https://samtools.github.io/hts-specs/VCFv4.2.pdf>`_.
+If the input tool you are using differs, it would be better to use a :ref:`custom conversion script <custom-conversion>`.
+
+**Assumptions on non-standard INFO fields**
+
+- ``PRECISE`` if given, Confidence intervals are ignored if given in favour of exact breakpoint calls using pos and END as the breakpoint positions
+- ``CT`` values if given are representative of the breakpoint orientations.
```

Comment:
Note: I'm quite sure 'CT' stands for 'connection type' in DELLY's documentation.  Can be useful when looking things up.

FYI - from https://groups.google.com/forum/#!topic/delly-users/6Mq2juBraRY
Hi Daniel,

These connection types (CT) are probably most relevant for complex re-arrangements in cancer. Another way to think about these is in terms of the paired-end orientation (Medvedev et al. 2010: PMID: 2080529, Figure 3). Different names exist for these connection types in the literature, e.g. Stephens et al. (PMID: 21215367) used head-to-head inversions, tail-to-tail inversions, and so on.

Intra-chromosomal rearrangements:
(1) deletion-type = tail-to-head = 3' to 5' = [+-]
(2) tandem-duplication type = head-to-tail = 5' to 3' = [-+]
(3) left-spanning inversion type = tail-to-tail = 3' to 3' = [++]
(4) right-spanning inversion type = head-to-head = 5' to 5' = [--]

For inter-chromosomal rearrangements the pairs can also map [+-], [-+], [++] and [--] just on 2 different chromosomes and hence, you have all 4 possible connections types for translocations in Delly. The easiest example is a single chromosome A where a virus B integrated. If B was integrated in forward orientation you will see a 3' to 5' translocation and a 5' to 3' translocation at the integration site. If B was integrated in inverted orientation you will see a 3' to 3' translocation and a 5' to 5' translocation at the integration site.

Best, Tobias

Reply:
I actually link to that very same google group in the code itself on my CT comments. But here I am talking about what they mean in terms of mavis conversions. We convert connection type (CT) to breakpoint orientations